docs/guides/deployment/fly_io.rst

```diff
@@ -169,7 +177,7 @@ running and then run ``edgedb instance link``:
 
 .. code-block:: bash
 
-   $ echo $EDGEDB_PASSWORD | edgedb instance link \
+   $ echo $PASSWORD | edgedb instance link \
         --trust-tls-cert \
         --host $EDB_APP.internal \
         --port 8080 \
```

Comment:
It would be great to add a section on how to use the instance from an app deployed to Fly.io, i.e. set `EDGEDB_HOST` to `myorg-edgedb.internal` (and `EDGEDB_CLIENT_TLS_SECURITY=insecure`)